examples/logrotate.cron

```diff
@@ -1,8 +1,23 @@
 #!/bin/sh
 
-/usr/sbin/logrotate /etc/logrotate.conf
-EXITVALUE=$?
-if [ $EXITVALUE != 0 ]; then
-    /usr/bin/logger -t logrotate "ALERT exited abnormally with [$EXITVALUE]"
+# exit immediately if there is another instance running
+if checkproc /usr/sbin/logrotate; then
```

Comment:
the whole example script seems distribution specific. perhaps another file if it's accepted to include it in upstream